src/angular/6-restaurant-service/restaurant-service.md

```diff
@@ -116,22 +116,22 @@ We've done some work to create a Place My Order API for use in this app by creat
 npm install place-my-order-api --save
 ```
 
-✏️ Next make add an api script to your `package.json`
+✏️ Next add an API script to your `package.json`
 
 ```js
   "scripts": {
     "ng": "ng",
     "start": "ng serve",
     "build": "ng build",
+    "watch": "ng build --watch --configuration development",
     "test": "ng test",
-    "lint": "ng lint",
```

Comment:
why are we removing lint and test commands?

Reply:
because this does not exist anymore in a new ng new command

Reply:
interestingly the e2e part is not created automatically anymore, and the lint script too